packages/icons/plugins/WalletUnderTabs.svg

```diff
@@ -1,21 +1,25 @@
 <svg viewBox="0 0 16 17" xmlns="http://www.w3.org/2000/svg">
-    <g xmlns="http://www.w3.org/2000/svg">
+    <g xmlns="http://www.w3.org/2000/svg" fill="currentColor">
```

Comment:
It's a duplicate of `plugins/ConnectWallet.svg`. Maybe we should use `ConnectWallet.svg` instead.